myhpom/templates/myhpom/accounts/next_steps.html

```diff
@@ -15,7 +15,7 @@ <h1 class="mx-auto h2 modal-title pseudo-modal__title pseudo-modal__title--cente
 
             <div class="pt-1 pb-1 modal-header modal-header--subtitle modal-header--next-steps pseudo-modal__header">
                 <div class="mx-auto mb-0 pseudo-modal__header-caption-text pseudo-modal__header-caption-text--next-steps">
-                    {% scribble 'next-steps-header' 'shared' %}
+                    {% scribble 'header-text' 'next_steps' %}
```

Comment:
For this scribble, since there is some text from the client lets put it here between the `{% scribble %}` and `{% endscribble %}` tags...keeping our styling in our general domain.

Reply:
Oh I guess this isn't actually possible if you want to create the entries in the db? I guess nevermind...djinni is out of the bottle, as it were.